src/Opserver.Poller/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Nest;
+using Opserver.Data;
+using Opserver.Poller.Services;
+
+namespace Opserver.Poller
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var serviceProvider = CreateServiceProvider();
+
+            var ps = serviceProvider.GetService<PollingService>();
+            await ps.StartAsync(new CancellationToken());
+            var sqlPoller = serviceProvider.GetService<IPollSql>();
+
+
+            Console.WriteLine("polling");
+            sqlPoller.ObserveAllInstances();
+
+            ConsoleKey cc;
+            do
+            {
+                cc = Console.ReadKey().Key;
+            } while (cc != ConsoleKey.Escape);
+        }
+
+        private static ServiceProvider CreateServiceProvider()
+        {
+            IConfiguration config = new ConfigurationBuilder()
+                .AddJsonFile("appSettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile("opserverPollerSettings.json", optional: false, reloadOnChange: true)
+                .AddJsonFile("localSettings.json", optional: true, reloadOnChange: true)
+                .Build();
```

Comment:
I think we should add environment variables here as well, so that we can drive the configuration via ConfigMaps in K8s by mapping them to env vars. This will make it easy to deploy in different pods with different connection strings, index paths etc. without having to make changes to the code base.